broker/src/main/java/org/apache/rocketmq/broker/processor/AdminBrokerProcessor.java

```diff
@@ -493,6 +490,17 @@ private synchronized RemotingCommand updateBrokerConfig(ChannelHandlerContext ct
         return response;
     }
 
+    private void applyUpdatedConfig(Properties properties) {
+        if (properties.containsKey(MixAll.DUP_NAMESRV_ADDR)) {
+            String updatedNamesrvAddr = this.brokerController.getBrokerConfig().getNamesrvAddr();
+            this.brokerController.getBrokerOuterAPI().updateNameServerAddressList(updatedNamesrvAddr);
```

Comment:
[QUESTION]  If `fetchNamesrvAddrByAddressServer` is true and namesrAddr is null when broker startup, broker will update namesrv address from address server periodically.  After you update it , the scheduled task will overwrite it at next period. Maybe you should cancle the scheduled task if it exist. Or you can return some tips if such update schedule task exist. I think remind user and let him decide whether or not update and cancle the task might be better.

Reply:
In current design, the priority of ```mqadmin updateBrokerConfig namesrvAddr``` is lower than ```fetchNamesrvAddrByAddressServer```, which means the scheduled task will overwrite it. The purpose of this PR is to enable dynamically effective namesrvAddr for name server auto-scaling when ```fetchNamesrvAddrByAddressServer``` scheduled task is not enabled. If the ```fetchNamesrvAddrByAddressServer``` scheduled task is enabled, it already have the ability of  dynamically effective.

Therefore, I don't think we should cancel the scheduled task.